### PR TITLE
MIGENG-139: WIR Workloads rule prevent websphere false positive

### PR DESCRIPTION
**Pull request by m-brophy**
https://jira.coreos.com/browse/MIGENG-139

### CODE REVIEW

**Review by jonathanvila — APPROVED** [review]

File: src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java [hunk 1]
```diff
@@ -1448,9 +1448,8 @@ public void testSQLServerOnWindows() {
         vmWorkloadInventoryModel.setVersion("6.5");
         vmWorkloadInventoryModel.setHost_name("esx13.v2v.bos.redhat.com");
 
-        List<String> vmDiskFilenames = new ArrayList<>();
         List<String> systemServicesNames = new ArrayList<>();
-        systemServicesNames.add("unix_service");
+        systemServicesNames.add("NOTwas");
```
Comment (jonathanvila) [inline src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java:1452]: what happens if there's no service, or if there is a service without "was" in its name ?
I mean to check the previous condition that was "contains" .

**Comment by PhilipCattanach** [comment]
I'm a little surprised to see the 'negative check' in the SQL Server on Windows test method.
However the logic and  function to check starts with look fine. And you do do have a test for the negative condition, so job done,

**Review by PhilipCattanach — APPROVED** [review]
As per my earlier comment - approved

File: src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java (hunk 1 above)
Comment (m-brophy) [reply src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java:1452]: There's already a method which checks what happens when there's no service - testNoServicesAndNoFiles().

In addition the intended purpose of the change to this line was to check that "was" no longer fires the rule if it's not at the start of a service, ie contains("was") is no longer a condition.

Hope that's okay.

**Review by mrizzi — APPROVED** [review]
Tested in CI with @PhilipCattanach and it worked nicely.
Thanks @m-brophy for the enhancement of the quality of the rule :+1:

**Pull request merged**